app/views/mailers/digest_mailer/digest_email.html.erb

```diff
@@ -5,17 +5,17 @@
 <ul>
   <% @articles.each do |article| %>
     <li style="margin:18px auto">
-      <a href="https://dev.to<%= article.path %>" style="font-weight: bold;color:#0045ff;font-size:calc(0.93em + 0.2vw);"><%= article.title.strip %></a> <%= truncate(article.description, length: 120) %>
+      <a href="<%= article_url(article) %>" style="font-weight: bold;color:#0045ff;font-size:calc(0.93em + 0.2vw);"><%= article.title.strip %></a> <%= truncate(article.description, length: 120) %>
     </li>
   <% end %>
      <hr />
 </ul>
 
 
 <div>
-  👋 <b>Looking for work?</b> <a href="https://dev.to/listings/jobs">Browse job openings</a> or create an <a href="https://dev.to/listings/forhire">available for hire</a> listing.
+  👋 <b>Looking for work?</b> <a href="<%= app_url("/listings/jobs") %>">Browse job openings</a> or create an <a href="<%= app_url("/listings/forhire") %>">available for hire</a> listing.
```

Comment:
unfortunately I couldn't find the correct helper to generate this listings by category path 🤷‍♂️